src/confluent_kafka/src/Consumer.c

```diff
@@ -980,6 +980,33 @@ static PyObject *Consumer_poll (Handle *self, PyObject *args,
 }
 
 
+static PyObject *Consumer_memberid (Handle *self, PyObject *args,
+                                    PyObject *kwargs) {
+        char *memberid = NULL;
+        PyObject *memberidobj;
+        CallState cs;
+        if (!self->rk) {
+                PyErr_SetString(PyExc_RuntimeError,
+                                "Consumer closed");
+                return NULL;
+        }
+
+        CallState_begin(self, &cs);
```

Comment:
memberid() is not a blocking function and does not trigger callbacks, so no need for a callstate.